addons/binding/org.openhab.binding.freemobilesms/src/main/java/fr/free/smsapi/Account.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package fr.free.smsapi;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * An interface for handling account's related data.
+ *
+ * @author Guilhem Bonnefille <guilhem.bonnefille@gmail.com> - Initial contribution
+ */
+@NonNullByDefault
+public interface Account {
```

Comment:
Very enterprisy to have an interface and an implementation but also very useless in this context.
Please unite interface+implementation+FreeMobileSmsConfiguration.

Reply:
I like to keep the fr.free.mobile clean from OpenHAB code as I reuse this code in other project. I should create a maven module.